.github/workflows/publish.yml

```diff
@@ -23,8 +23,8 @@ jobs:
       run: pip install -r requirements.txt;
     - name: script
       run: nosetests --exclude=.*
-    - name: build dist
-      run: python setup.py bdist
+    - name: package dist
+      run: python setup.py sdist
```

Comment:
@bryanateoan found this post https://medium.com/ochrona/understanding-python-package-distribution-types-25d53308a9a at the end it recommends to create both types, I'm not a python expert so we may want to ask python agent team for guidance.

Additional readings:
- https://dev.to/icncsx/python-packaging-sdist-vs-bdist-5ekb
- https://devopedia.org/python-distributions-and-packaging

Reply:
it looks like python agent is using the 2 types of distribution and the 2 modes of bdist
https://github.com/Contrast-Security-Inc/python-agent/tree/develop/scripts